lib/services/elasticsearch.js

```diff
@@ -849,7 +861,7 @@ class ElasticSearch extends Service {
         item = esRequest.body[i],
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') "item" hides or potentially hides a variable declared in an outer scope at line 1365. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AVariableShadowing)